src/averageAges.js

```diff
@@ -20,6 +20,22 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const manWithCentery = people.filter(item => {
+    return Math.ceil(item.died / 100) === century && item.sex === 'm';
+  });
+
+  const manWithoutCentery = people.filter(item => item.sex === 'm');
+
+  const avAgeWithCent = manWithCentery.reduce((a, b) => {
+    return a + (b.died - b.born);
+  }, 0) / manWithCentery.length;
+
+  const avAgeWithoutCent = manWithoutCentery.reduce((a, b) => {
+    return a + (b.died - b.born);
+  }, 0) / manWithoutCentery.length;
```

Comment:
move it in the separate method

Reply:
What do you meens?

Reply:
@tetiana-ivanchenko What do you meen?

Reply:
I mean: move the code that counts averageAge into the separate function.
Actually, if you have any questions: feel free to ask it in the fe_chat2